Findhttpxx.cmake

```diff
@@ -0,0 +1,98 @@
+#
```

Comment:
No, and it's not easily done. Was my first thought too. Maybe in a second iteration.